portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -50,7 +50,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     int numIteratedComments = 0;
     for (Entity entity : results.asIterable()) {
       long id = entity.getKey().getId();
-      String comment = (String) entity.getProperty("comment");
+      String username;
+      if (entity.getProperty("username") == null) {
```

Comment:
Instead of duplicating these property/entity names inline, prefer to use String constants at the class level, e.g.

```java
private static final String ENTITY_COMMENT = "Comment";
private static final String PROPERTY_COMMENT = "comment";
private static final String PROPERTY_USERNAME = "username";
```

This ensures typos are caught at compile-time, and ensures that any changes update all occurrences at once.